tools/code_format/BUILD

```diff
@@ -10,13 +10,11 @@ exports_files([
     "check_format.py",
     "header_order.py",
     "envoy_build_fixer.py",
-    "flake8.conf",
 ])
 
 py_binary(
     name = "python_flake8",
     srcs = ["python_flake8.py"],
-    data = [":flake8.conf"],
```

Comment:
this file is moved to the root of the repo so it is automatically picked up running the tool